src/main/java/frc/robot/common/ControlPanelColor.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.common;
+
+import edu.wpi.first.wpilibj.util.Color;
+
+public enum ControlPanelColor {
```

Comment:
Move this to its own class, since it's not directly linked to the sensor. In fact, the sensor can't deal with ControlPanelColors, it can only deal with wpilib Colors.